com.unity.template-universal/Packages/manifest.json

```diff
@@ -2,19 +2,20 @@
   "dependencies": {
     "com.unity.2d.sprite": "1.0.0",
     "com.unity.2d.tilemap": "1.0.0",
-    "com.unity.ads": "3.3.1",
-    "com.unity.analytics": "3.3.5",
-    "com.unity.collab-proxy": "1.3.7",
-    "com.unity.ext.nunit": "1.0.0",
-    "com.unity.ide.rider": "1.2.1",
-    "com.unity.ide.visualstudio": "2.0.0",
-    "com.unity.ide.vscode": "1.1.4",
-    "com.unity.purchasing": "2.0.6",
+    "com.unity.ads": "3.7.5",
+    "com.unity.analytics": "3.6.11",
+    "com.unity.collab-proxy": "1.9.0",
+    "com.unity.ext.nunit": "1.0.6",
+    "com.unity.ide.rider": "3.0.7",
+    "com.unity.ide.visualstudio": "2.0.11",
+    "com.unity.ide.vscode": "1.2.4",
+    "com.unity.purchasing": "3.2.2",
     "com.unity.render-pipelines.universal": "12.0.0",
-    "com.unity.test-framework": "1.1.18",
-    "com.unity.textmeshpro": "3.0.0-preview.4",
-    "com.unity.timeline": "1.2.11",
+    "com.unity.test-framework": "1.1.29",
+    "com.unity.textmeshpro": "3.0.6",
+    "com.unity.timeline": "1.6.2",
     "com.unity.ugui": "1.0.0",
+    "com.unity.xr.legacyinputhelpers": "2.1.8",
```

Comment:
This seems different than what the original PR has...
```
    "com.unity.ads": "3.7.1",
    "com.unity.analytics": "3.6.11",
    "com.unity.collab-proxy": "1.7.1",
    "com.unity.ext.nunit": "1.0.6",
    "com.unity.ide.rider": "3.0.7",
    "com.unity.ide.visualstudio": "2.0.9",
    "com.unity.ide.vscode": "1.2.3",
    "com.unity.purchasing": "3.2.2",
    "com.unity.render-pipelines.universal": "12.0.0",
    "com.unity.test-framework": "1.1.27",
    "com.unity.textmeshpro": "3.0.6",
    "com.unity.timeline": "1.6.1",
    "com.unity.xr.legacyinputhelpers": "2.1.8",
```

Reply:
These version numbers get automatically bumped when opening the project. I'm going to keep as is as these are more recent number and because the numbers, in any case, can become dated quickly.